website/src/components/IIGO/IIGO.module.css

```diff
@@ -0,0 +1,33 @@
+.text {
+    font-size: 2em;
+    font-weight: 600;
+}
+
+.root {
+    margin: 0;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    -ms-transform: translate(-50%, -50%);
+    transform: translate(-50%, -50%);
+}
+
+.appLogo {
+    height: 40vmin;
+    pointer-events: none;
+}
+
+@media (prefers-reduced-motion: no-preference) {
+    .appLogo {
+        animation: App-logo-spin infinite 20s linear;
+    }
+}
+
+@keyframes App-logo-spin {
+    from {
+        transform: rotate(0deg);
+    }
+    to {
+        transform: rotate(360deg);
+    }
+}
```

Comment:
the spinning app logo is in every file?

also if the types are shared globally just put them in a global (non-module) css file

styles shared between modules can be put in a folder root or something.